File: wireless/commands/list-example-1/list-example-1.py
# Download the **Next-Gen** twilio-python library from:
# twilio.com/docs/libraries/python#installation-nextgen
from twilio.rest import Client

account_sid = "{{ account_sid }}" # Your Account SID from www.twilio.com/console
auth_token  = "{{ auth_token }}"  # Your Auth Token from www.twilio.com/console

client = Client(account_sid, auth_token)
wireless = client.preview.wireless

for cmd in wireless.commands.list():
    print(cmd.command)

File: wireless/rateplans/list-example-1/list-example-1.rb
# Download the **Next-Gen** twilio-ruby library from:
# twilio.com/docs/libraries/ruby#installation-nextgen
require 'twilio-ruby'

account_sid = '{{ account_sid }}'
auth_token = '{{ auth_token }}'

@client = Twilio::REST::Client.new account_sid, auth_token

@client.preview.wireless.rate_plans.list().each do |plan|
  puts plan.sid
  puts plan.alias_
end